kickoff/static/suggestions.js

```diff
@@ -121,16 +125,46 @@ function populatePartial(name, version, previousBuilds, partialElement) {
         } else {
             previousReleases = previousBuilds[base + 'release'].sort().reverse()
         }
+
+        if (isESR(version)) {
+            // Use the ESR partial
+            partialsADI = partialsADIESR;
+        } else {
+            partialsADI = partialsADIRelease;
+        }
+        // For thunderbird, use only the three last
+
         nbPartial = 3;
     }
 
+    // Transform the partialsADI datastruct in a single array to
+    // simplify processing
+    partialsADIVersion = [];
+    for (i = 0; i < partialsADI.length; i++) {
+        partialsADIVersion[i] = partialsADI[i].version;
+    }
+
     partial = "";
     partialAdded = 0;
     for (i = 0; i < previousReleases.length; i++) {
+
         // Reuse the previous builds info to generate the partial
         // Strip the build number. "31.0build1" < "31.0.1" => false is JS
         previousRelease = previousReleases[i].replace(/build.*/g, "");
-        if (previousRelease < version) {
+
+        // Some explanations on this code
+        // We want:
+        // * the last version published (no matter the ADI).
+        //   This because the ADI takes some time to be relevant
+        // * we take the versions which match the highest ADIs
+        // * For TB, uses only the X last builds
```

Comment:
Are we using the last N builds for Thunderbird because we don't have ADI for them?